src/Microsoft.ML.Data/Scorers/PredictionTransformer.cs

```diff
@@ -500,13 +517,15 @@ private protected override void SaveCore(ModelSaveContext ctx)
             base.SaveCore(ctx);
 
             ctx.SaveStringOrNull(_trainLabelColumn);
+            ctx.SaveStringOrNull(_scoreColumn);
+            ctx.SaveStringOrNull(_predictedLabelColumn);
         }
 
         private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
                 modelSignature: "MC  PRED",
-                verWrittenCur: 0x00010001, // Initial
+                verWrittenCur: 0x00010002, // Initial
```

Comment:
Typically the way this is written is like the following:

[https://github.com/dotnet/machinelearning/blob/7cb98932523774cf988c88a825c516a11c216444/src/Microsoft.ML.Transforms/Text/TokenizingByCharacters.cs#L79-L80](https://github.com/dotnet/machinelearning/blob/7cb98932523774cf988c88a825c516a11c216444/src/Microsoft.ML.Transforms/Text/TokenizingByCharacters.cs#L79-L80)

That way we see each version, and what was changed in each version.